index.html

```diff
@@ -50,7 +51,258 @@ <h3 class="header-highlight-2">I’am a Software Developer</h3>
+    </div>
+    <section class="about-section">
+      <div class="about-header">
+        <h1>About me</h1>
+        <p class="about-header-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <button type="button" class="button">get my resume</button>
+
+        <hr />
+      </div>
+      <div class="skills">
+        <div class="language-contaier">
+          <div class="diamond-language">
+            <span class="diamond"></span>
+            <span class="language">language</span>
+          </div>
+          <ul class="lang">
+            <li class="lang-item java">
+              <a href="#" class="java-text">javascript</a>
+            </li>
+            <li class="lang-item java">
+              <a href="#" class="java-text">ruby</a>
+            </li>
+            <li class="lang-item java">
+              <a href="#" class="java-text">html</a>
+            </li>
+            <li class="lang-item java">
+              <a href="#" class="java-text">css</a>
+            </li>
+          </ul>
+        </div>
+        <div class="language-contaier">
+          <div class="diamond-language">
+            <span class="square"></span>
+            <span class="language">Framework</span>
+          </div>
+          <ul class="lang">
+            <li class="lang-item java">
+              <a href="#" class="java-text">bootstrap</a>
+            </li>
+            <li class="lang-item java">
+              <a href="#" class="java-text">ruby</a>
+            </li>
+            <li class="lang-item java">
+              <a href="#" class="java-text">RSPec</a>
+            </li>
+            <li class="lang-item java">
+              <a href="#" class="java-text">CapyBara</a>
+            </li>
+            <li class="lang-item java">
+              <a href="#" class="java-text">Selenium</a>
+            </li>
+          </ul>
+        </div>
+        <div class="language-contaier">
+          <div class="diamond-language">
+            <span class="circle"></span>
+            <span class="language">skills</span>
+          </div>
+          <ul class="lang">
+            <li class="lang-item java">
+              <a href="#" class="java-text">codekit</a>
+            </li>
+            <li class="lang-item java">
+              <a href="#" class="java-text">github</a>
+            </li>
+            <li class="lang-item java">
+              <a href="#" class="java-text">codepen</a>
+            </li>
+            <li class="lang-item java">
+              <a href="#" class="java-text">gitlab</a>
+            </li>
+            <li class="lang-item java">
+              <a href="#" class="java-text">terminal</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Kindly fix the space that is showing after the about me section. 
Here you have a screenshot of how I am seeing it:
![image](https://user-images.githubusercontent.com/74506933/153713572-c7f8a787-d933-42ed-8bd7-cc72c448174b.png)